setup-mcp.ps1

```
@@ -0,0 +1,260 @@
+    
+    Set-Location $McpServerPath
+    dotnet run
+}
+
+function Test-McpServer {
+    Write-Host ""
+    Write-Host "Testing MCP Server..." -ForegroundColor Green
+    Write-Host ""
+    
+    # Test connection
+    Write-Host "Testing connection to http://localhost:5001/mcp..." -ForegroundColor Yellow
```

PSScriptAnalyzer (1.24.0)

[warning] File 'setup-mcp.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)